deploy/crds/kieapp.crd.v1beta1.yaml

```diff
@@ -1321,7 +1321,7 @@ spec:
                     properties:
                       url:
                         type: string
-                        description: LDAP Endpoint to connect for authentication
+                        description: LDAP endpoint to connect for authentication. For failover set two or more LDAP endpoints separated by space.
```

Comment:
this change is unnecessary.

Reply:
right, there is two occurrences on the same file. updated on the first one.

Reply:
updated